packages/cli/src/deflake-tests/deflake-tests.handler.ts

```diff
@@ -1,8 +1,5 @@
 import { METICULOUS_LOGGER_NAME } from "@alwaysmeticulous/common";
-import {
-  ReplayExecutionOptions,
-  ReplayTarget,
-} from "@alwaysmeticulous/common/dist/types/replay.types";
+import { ReplayExecutionOptions, ReplayTarget } from "@alwaysmeticulous/common";
```

Comment:
You'll need to combine this with import above for lint to not fail. You can use Command + Shift + P -> Optimize Imports in vscode, or just merge in develop and hit save and the auto-fix should solve it.